prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html

```diff
@@ -1,42 +1,101 @@
-<div class="container-fluid">
-
-  <div class="row">
-    <div class="col-sm-7 col-md-6 col-lg-5 col-xl-3">
-
-      <div class="d-flex flex-column align-items-center">
-
-        <img src="/assets/prime_logo.svg"
-             class="logo"
-             alt="PRIME Logo">
-
-        <div class="action">
-          <button mat-flat-button
+<div>
+  <div class="row mt-4 enrolment-container">
+    <div class="col-md-3 col-sm-12">
+      <img src="/assets/prime-logo-textless.svg"
+           height="200"
+           width="200"
+           class="logo"
+           alt="PRIME Logo">
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <span class="large-title mb-8"><strong>PRIME</strong></span>
+      <h2 class="mb-8"><strong>PharmaNet Enrolment</strong></h2>
+      <div class="ml-2">
+        <p class="h3 mb-0"><strong>As a Health Professional working in British Columbia, use PRIME to request approval from The
+        Ministry
+        of
+        Health
+        to access PharmaNet.</strong></p>
+        <p class="lead mb-4">PRIME provides approval for access to PharmaNet, but not access itself.</p>
+        <p class="lead mb-4 accent">To get started you will need:</p>
+        <span class="h3"><strong>BC Services Card, Mobile Device, and College Certification Information</strong>
+        </span>
+        <span class="lead ml-2">(for prescribers only)</span>
+        <div class="mt-4"><button mat-flat-button
+                  class="mr-4"
                   color="primary"
-                  (click)="loginUsingBCSC()">Log-in with your BC Services Card</button>
+                  (click)="loginUsingBCSC()">Log in with BC Services Card</button>
+          <button mat-flat-button
+                  color="accent"
+                  href="#">Learn More</button></div>
```

Comment:
Do we have any information on where this button should go?

Reply:
Currently we do not. Will make a ticket to note these need to be linked up.